### DISCUSSION TITLE
Ability to remove 'Unread Mention' status from all splits/tabs with the same channel

### DISCUSSION THREAD

**Discussion by RyanPotat** [Chatterino/chatterino2#4547] [Ideas]
I have quite a few tabs with multiple splits, and many of the channels overlap. I also have some of those channels as their own independent tab. 

So, with specific channels, if I receive a highlight, it will mark 4-5 tabs with an unread highlight. 

![image](https://user-images.githubusercontent.com/127474225/232265810-65a8d140-3d0c-422d-9c18-5d26206e2fb4.png)
I was @'ed in the same single channel and:
![image](https://user-images.githubusercontent.com/127474225/232265823-d11a5479-f569-4e2b-a0a6-c1bc820b41e0.png)
Since I had the channel currently open, and read the highlighted mention, having it highlight other tabs is redundant.

My suggestion is when you read a highlight in a single channel, it removes the unread (mentioned/highlighted) status for that message across all splits/tabs containing that channel. 

Thanks!

**Comment by leon-richardt** [comment]
Thanks for the report. Would you agree that the problem and suggested solution described here is covered sufficiently in https://github.com/Chatterino/chatterino2/issues/1703?

**Reply by RyanPotat** [reply]
Yes it looks like that basically requests the same feature I would like to be added. Sorry for not seeing that one.
